### ISSUE TITLE
Problem audio megane 3 rs 

### ISSUE THREAD

**Issue by Gaspaar13** [cedricp/ddt4all#900]
Hello, after being reprogram with clip
Something went wrong, don’t have values in acoustic and can’t write (wrong response) 
With Renault clio can’t reprogramar again because said that is already updated, this radio can be virginized? Or maybe save it with ddt?

thanks

**Comment by shrlnm** [comment]
Hello @Gaspaar13 
what was reprogrammed ? LGU? MICOM?  from which version to which?

**Comment by matzel82** [comment]
Do you read before/after steps ??
Are you a Renault Staff ?

**Comment by Gaspaar13** [comment]
> from

Reprog audio, works fine (completed), but when I try again (looking to install version from a newer megane to get rs monitor) didn't do anything and said complete ![image](https://github.com/cedricp/ddt4all/assets/118029866/733fba11-854e-4933-8eaa-0ba31066d1a3)

**Comment by matzel82** [comment]
So...you need access to the Renault Server.... you need a USB Token.

**Comment by Gaspaar13** [comment]
> So...you need access to the Renault Server.... you need a USB Token.

Only official usb or there is another way?

**Comment by matzel82** [comment]
> > Do you read before/after steps ??
> > Are you a Renault Staff ?
> 
> Read yea 
> No, I'm not a Renault staff 

Normally it's prohibited to use the software. The Software calls at Home and transferred the VIN to Renault. The know where you are...

**Comment by matzel82** [comment]
Especially... you can't get software from another car to yours....only with your vin connected "versions"

**Comment by Gaspaar13** [comment]
Yes I think the same but worth the try ![image](https://github.com/cedricp/ddt4all/assets/118029866/a1ccfd12-946e-440a-9a74-75698ffe8652)

Only need the missing script, it's the same radio and miu

**Comment by Gaspaar13** [comment]
Maybe virginize?
